### ISSUE TITLE
Add a unit field to the `RegulatedProduct` field

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-reporting#369]
#### Description:

As an industrial reporter, I want to know which unit to report my production in, so that I can accurately fill out my report

We need a "unit" field on the `RegulatedProduct` model, and the data can be found in GGERR Schedule A.1

keywords: units product products

reference for the units is in the "Dropdown" tab of [this file](https://bcgov.sharepoint.com/:x:/r/teams/00608-CPRInternal/Shared%20Documents/Carbon%20Pricing%20Transition%20SMEs/Registration%20App/Registration%20Pt.%202%20Fields.xlsx?d=w8cf871009f8d43c19d1d79c69fa259e8&csf=1&web=1&e=NdGXiC)

#### Acceptance Criteria:

Given I am a developer
When I read a RegulatedProduct object from the database
Then I have access to a 'unit' field containing the unit in which this product is measured

Given I am an industrial reporter
When I report production
Then the form displays the unit for the product selected

Given I am an IRC analyst
When I look at the data

Blocked by #203 

#### Development Checklist:

- [x] a 'Unit' field (varchar 1000) on the `RegulatedProduct` table
- [x] a 'regulated' field (boolean) on the `RegulatedProduct` table
- [x] the unit is displayed in the production data form
- [x] Units are pre-populated in the migration
- [x] Update the production data form to pull the unit from the db
- [x] tests
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by pbastia** [comment]
@patriciarussellCAS the units of production reported will be "tonnes" until this is done

**Comment by rdromey** [comment]
@pbastia to write AC
